documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -53,10 +53,10 @@ OAUTHBEARER authentication can only be used by Kafka clients that support the OA
 
 .PLAIN overview
 
-PLAIN is a simple authentication mechanism used by all Kafka client tools (including developer tools such as kafkacat). 
-To enable PLAIN to be used together with OAuth 2.0 authentication, Strimzi includes server-side callbacks and calls this _OAuth 2.0 over PLAIN_. 
+PLAIN is a simple authentication mechanism used by all Kafka client tools (including developer tools such as kcat). 
```

Comment:
Do we explain somewhere what `kcat` / `kafkacat` is? If not, we should maybe either include a link or just remove it.

Reply:
Simpler to delete